dockerfiles/image_hash

```diff
@@ -0,0 +1,2 @@
+# sha256 digest quay.io/freedomofpress/packaging-debian-buster:2021_07_09
+7ac0e1e1c29d9a60e210e0da246a6d60e49c9eab18cf654bacf95ce5fed1413b
```

Comment:
shouldn't this be: `5e1d3cf73ac6bfe418d499fc419bf789a0d365c51c157b704969d88d2df25e65`? also i'm seeing https://security-tracker.debian.org/tracker/CVE-2019-19814 for this image. trying to see if we have it documented somewhere that this is fine.

Reply:
>shouldn't this be: 5e1d3cf73ac6bfe418d499fc419bf789a0d365c51c157b704969d88d2df25e65? also i'm seeing https://security-tracker.debian.org/tracker/CVE-2019-19814 for this image. trying to see if we have it documented somewhere that this is fine.

I actually have similar question and forgot to put in the comment. The checksum mentioned here is created by our scripts, may be I did a mistake in calling the script. Thank you for pointing out.

Reply:
It is now fixed with the hash of the newer image.

Reply:
great I'll take another look at this PR